Android App/app/src/main/java/org/cfp/citizenconnect/Feedback/FragmentFeedback.java

```diff
@@ -88,6 +93,28 @@ public View onCreateView(LayoutInflater inflater, @Nullable ViewGroup container,
         return rootView;
     }
 
+    @Override
+    public void onStart() {
+        super.onStart();
+        EventBus.getDefault().register(this);
+    }
+
+    @Override
+    public void onStop() {
+        EventBus.getDefault().unregister(this);
+        super.onStop();
+    }
+
+    @Subscribe(threadMode = ThreadMode.MAIN)
+    public void onMessageEvent(MessageEvent event) {
+        if(event.status){
+            chooseAccount();
+        }
+        else {
```

Comment:
if there's nothing in `else {}` then better remove it

Reply:
minor but agreed